usaspending_api/spending_explorer/v2/filters/spending_filter.py

```diff
@@ -64,10 +64,7 @@ def spending_filter(alt_set, queryset, filters, _type):
 
             # recipient
             elif key == "recipient":
-                alt_set = alt_set.filter(
-                    Q(award__latest_transaction__assistance_data__awardee_or_recipient_legal=value)
-                    | Q(award__latest_transaction__contract_data__awardee_or_recipient_legal=value)
-                )
+                alt_set = alt_set.filter(Q(award__latest_transaction_search__recipient_name_raw=value))
```

Comment:
As your story told you to do, I see you updated to `latest_transaction_search`, but why did you change this query from what it used to do (beyond the `latest_transaction_search` update)?

Reply:
`latest_transaction__assistance_data` and `latest_transaction__contract_data` are both views on top of `transaction_search`. One is a view where `is_fpds = False` and the other view where `is_fpds = True`. Since the `is_fpds` column isn't null-able, there's no reason to join to the `transaction_search` twice like we were before.